elements/lisk-cryptography/src/convert.ts

```diff
@@ -80,57 +80,105 @@ export const stringifyEncryptedPassphrase = (
 	if (typeof encryptedPassphrase !== 'object' || encryptedPassphrase === null) {
 		throw new Error('Encrypted passphrase to stringify must be an object.');
 	}
-	const objectToStringify = encryptedPassphrase.iterations
-		? encryptedPassphrase
-		: {
-				salt: encryptedPassphrase.salt,
-				cipherText: encryptedPassphrase.cipherText,
-				iv: encryptedPassphrase.iv,
-				tag: encryptedPassphrase.tag,
-				version: encryptedPassphrase.version,
-		  };
+	const objectToStringify = {
+		kdf: encryptedPassphrase.kdf,
+		cipher: encryptedPassphrase.cipher,
+		version: encryptedPassphrase.version,
+		ciphertext: encryptedPassphrase.ciphertext,
+		mac: encryptedPassphrase.mac,
+		salt: encryptedPassphrase.kdfparams.salt,
+		iv: encryptedPassphrase.cipherparams.iv,
+		tag: encryptedPassphrase.cipherparams.tag,
+		iterations: encryptedPassphrase.kdfparams.iterations,
+		parallelism: encryptedPassphrase.kdfparams.parallelism,
+		memorySize: encryptedPassphrase.kdfparams.memorySize,
+	};
 
 	return querystring.stringify(objectToStringify);
 };
 
-const parseIterations = (iterationsString?: string): number | undefined => {
-	const iterations = iterationsString === undefined ? undefined : parseInt(iterationsString, 10);
+const parseOption = (optionString?: string): number | undefined => {
```

Comment:
I kept it individual in case in the future new options appear, we would have to update that function. For now I think it's sufficient to keep it as it is.